sdk/webpubsub/azure-messaging-webpubsubservice/README.md

```diff
@@ -43,14 +43,26 @@ python -m pip install azure-messaging-webpubsubservice
 
 ### Authenticating the client
 
-In order to interact with the Azure WebPubSub service, you'll need to create an instance of the [WebPubSubServiceClient][webpubsubservice_client_class] class. In order to authenticate against the service, you need to pass in an AzureKeyCredential instance with endpoint and api key. The endpoint and api key can be found on the azure portal.
+To use an [Azure Active Directory (AAD) token credential][authenticate_with_token],
```

Comment:
@msyyc I would add multiple sections for authenticating the client, since there are at least 2 ways of authenticating. [Here's](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/textanalytics/azure-ai-textanalytics#authenticate-the-client) an example readme for a client that you can authenticate to with `AzureKeyCredential` and `AAD`